File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        merged_index=len(nums1)-1
        for value in reversed(nums2):
            nums1[merged_index]=value
            merged_index-=1
        nums1.sort()



    


        

File: 0088-merge-sorted-array/0088-merge-sorted-array.ts
/**
 Do not return anything, modify nums1 in-place instead.
 */
function merge(nums1: number[], m: number, nums2: number[], n: number): void {

    let merged_index=nums1.length-1;

    for(let i=nums2.length-1;i>=0;i--){
        nums1[merged_index]=nums2[i]
        merged_index-=1
    }
    nums1.sort((a,b)=>a-b)



    
};